server/src/main/java/org/opensearch/search/query/QuerySearchResult.java

```diff
@@ -370,6 +370,7 @@ public void readFromWithId(ShardSearchContextId id, StreamInput in) throws IOExc
 
     @Override
     public void writeTo(StreamOutput out) throws IOException {
+        super.writeTo(out);
```

Comment:
Why do we need it?

Reply:
revert, but I guess it makes sense to have this